frontend/projects/upgrade/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.scss

```diff
@@ -243,9 +243,11 @@ $font-size-small: 15px;
 
           span {
             margin: 5px;
-            border-radius: 14px;
-            background-color: rgba(216, 216, 216, 0.6);
-            padding: 7px 10px;
+            border-radius: 16px;
```

Comment:
i know it's an existing rule but if these spans are all in the template this could be nicer as a named class also, while we're at it.

Reply:
Thanks! I've wrapped the inner text of `mat-chip` with `span` and added a `chip-label` class to these span elements (just like how I did for styling checkbox labels)

Reply:
I've also updated the fake chips in the experiment view page to use `mat-chip` so I could get rid of those styles.